object_detection_tracking/src/object_detection_tracking_worker.cpp

```diff
@@ -42,16 +57,34 @@ void ObjectDetectionTrackingWorker::detectedObjectCallback(const autoware_msgs::
     obj.presence_vector = obj.presence_vector | obj.SIZE_PRESENCE_VECTOR;
     obj.presence_vector = obj.presence_vector | obj.OBJECT_TYPE_PRESENCE_VECTOR;
     obj.presence_vector = obj.presence_vector | obj.DYNAMIC_OBJ_PRESENCE;
-
+    
     // Object id. Matching ids on a topic should refer to the same object within some time period, expanded
     obj.id = obj_array.objects[i].id;
 
     // Pose of the object within the frame specified in header
+
     obj.pose.pose = obj_array.objects[i].pose;
+
+    obj.pose.pose.position.x = obj_array.objects[i].pose.position.x + velodyne_transform.getOrigin().getX();
+
+    obj.pose.pose.position.y = obj_array.objects[i].pose.position.y + velodyne_transform.getOrigin().getY();
+
+    ROS_WARN_STREAM(obj.pose.pose.position.x);
+
+    ROS_WARN_STREAM(obj.pose.pose.position.y);
```

Comment:
you can remove lines 66 - 74 since the  pose is being set in the doTransform call